File: java/Easy/Fibonacci-Number.java
class Solution {
    public int fib(int n) {
        // base condition
        // if the number is less than two return the number
        // because the fibonacci series starts from 
        // 0 1 so if n is 0 return 0
        // if n is 1 return 1 
            if( n < 2 ) {
                return n;
            }

            //calculating the fib number 
            // return the answer 
        return fib( n -1 ) + fib( n - 2 );
    }
}

File: C/Fibonacci-Number.c
int fib(int n){
    int n1 = 0;
    int n2 = 1;
    int n3 = n1+n2;
    if( n == 0 )
      return n;
      else {
for( int k=2; k<n; k++ ) {
    n1 = n2;
    n2 = n3;
    n3 = n1 + n2;
    
}
       

return n3;
     
      }
      return NULL;
}
